portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,132 @@
+	      rested I know he will carry his team to another ring.
+      </p>
+    </div>
+
+    <h2>Music</h2>
+    
+    <div class = "hobby">
+      <h4>What I Listen To</h4>
+      <p>
+          I am always open to listen to any new genres of music, any new artists or any new 
+          instruments. The most important thing for me to like a song or sound is that I
+          need to be able to feel the emotion that the artist puts into it. Without emotion,
+          music is just a bunch of nonsensical sounds to me. The emotion is what makes it an
+          art and is the reason why not everyone can be an artist (completely my own opinion).
+      </p>
+      <p>
+          With that being said, here is a list of my favorite artists/bands:
+          <br><br>
+          - Michael Jackson<br>
+          - Mac Miller<br>
+          - Key!<br>
+          - Trippie Redd<br>
+          - Kodak Black<br>
+          - Maroon 5<br>
+          - Queen<br>
+          - Kid Cudi<br>
+          - Thundercat<br>
+          - XXXTENTACION<br>
+          <br>
+          There are too many more for me to keep going, but I think you get the idea.
+      </p>
+    </div>
+
+    <div class = "hobby">
+      <h4>Instruments</h4>
+      <p>
+          I would love to learn as many instruments as possible. A dream of mine for when I have my
+          own place is to be able to have an entire room filled with instruments, and at any given
+          time be able to pick up an instrument and play it like a boss. I don't know what it is,
+          but instruments are just so fascinating to me.
+      </p>
+      <p>
+          The first musical instrument I learned to play was the violin. I started playing in fourth
+          grade and continued playing until I graduated high school. I went to two competitions with
+          my high school orchestra and won first place both times. While I was playing I didn't think
+          it would be something that I missed (mostly because I did not like my music teacher), but
+          after not playing in years I do miss it a lot. Missing the violin, however, led me to pick
+          up the guitar. I am now currently learning it and am still very new to the guitar, but I
+          hope to get better and better.
+      </p>
+      <p>
+          I also learned the piano when I was younger, but I stopped playing for a few years and I
+          have now forgotten everything I had learned. I don't own a piano anymore, but I have a
+          MIDI Keyboard that I connect to my computer and mess around with in FL Studio from time
+          to time. FL Studio, for those unaware, is a music production software and I use it to
+          make songs. Although I like to make my own songs, I have not released anything to the
+          public. I make music for myself, but maybe one of these days I will share what I have.
+      </p>
+    </div>
```

Comment:
```suggestion
    <div class = "hobby">
      <h4>Instruments</h4>
      <p>
        I would love to learn as many instruments as possible. A dream of mine for when I have my
        own place is to be able to have an entire room filled with instruments, and at any given
        time be able to pick up an instrument and play it like a boss. I don't know what it is,
        but instruments are just so fascinating to me.
      </p>
      <p>
        The first musical instrument I learned to play was the violin. I started playing in fourth
        grade and continued playing until I graduated high school. I went to two competitions with
        my high school orchestra and won first place both times. While I was playing I didn't think
        it would be something that I missed (mostly because I did not like my music teacher), but
        after not playing in years I do miss it a lot. Missing the violin, however, led me to pick
        up the guitar. I am now currently learning it and am still very new to the guitar, but I
        hope to get better and better.
      </p>
      <p>
        I also learned the piano when I was younger, but I stopped playing for a few years and I
        have now forgotten everything I had learned. I don't own a piano anymore, but I have a
        MIDI Keyboard that I connect to my computer and mess around with in FL Studio from time
        to time. FL Studio, for those unaware, is a music production software and I use it to
        make songs. Although I like to make my own songs, I have not released anything to the
        public. I make music for myself, but maybe one of these days I will share what I have.
      </p>
    </div>
```